native/jsonrs/Cargo.toml

```
@@ -11,9 +11,9 @@ crate-type = ["dylib"]
 
 [dependencies]
 flate2 = "1.0"
-rustler = "0.28.0"
+rustler = "0.30.0"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
-serde_rustler = { git = "https://github.com/benhaney/serde_rustler", tag = "v0.0.7" }
+serde_rustler = { git = "https://github.com/lytedev/serde_rustler", rev = "cd7c66fbc837135616a7ff39c13623de65e9f56f" }
```

serde_rustler is tagged now
```suggestion
serde_rustler = { git = "https://github.com/benhaney/serde_rustler", tag = "v0.0.8" }
```
plus Cargo.lock change